src/redux/categories/catagoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = 'Under Constructions';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => initialState,
+  },
```

Comment:
Kindly for the check status action return a string "Under construction" and In case of unknown action - return the current state.